frontend/package.json

```diff
@@ -36,6 +36,7 @@
     "ember-cli-surge": "^1.1.20",
     "ember-cli-template-lint": "^0.7.3",
     "ember-cli-uglify": "^1.2.0",
+    "ember-cli-underscore": "0.0.3",
```

Comment:
Use lodash instead please .. 
https://github.com/mike-north/ember-lodash 

Has the same functions as underscore and more and is better performance wise.